astropy/time/core.py

```diff
@@ -1633,6 +1634,26 @@ def max(self, axis=None, out=None, keepdims=False):
             )
         return self[self._advanced_index(self.argmax(axis), axis, keepdims)]
 
+    def _ptp_impl(self, axis=None, out=None, keepdims=False):
+        if out is not None:
+            raise ValueError(
+                "Since `Time` instances are immutable, ``out`` "
+                "cannot be set to anything but ``None``."
+            )
+        return self.max(axis, keepdims=keepdims) - self.min(axis, keepdims=keepdims)
+
+    if NUMPY_LT_2_0:
+        _ptp_decorator = lambda f: f
+    else:
+        _ptp_decorator = deprecated("6.1", alternative="np.ptp")
```

Comment:
`np` is just a short-form, so should use real package import name?

```suggestion
        _ptp_decorator = deprecated("6.1", alternative="numpy.ptp")
```

Reply:
I really cannot understand what Past Self was doing. 😅

Reply:
Is there a standard practice for such things?

Reply:
It looks like Numpy itself uses the short form more often than not in their own release notes, but then again not systematically, so maybe we shouldn't overthink it.
But again, if you think the short form might confuse anyone I'm happy to just use the full name !

Reply:
Naw, if you are following precedent, just leave it. Thanks!